packages/api-core/src/constants.ts

```diff
@@ -250,3 +253,9 @@ export const getDataCenterOptions = (): Record< DataCenterOption, string > => (
 	dca: __( 'US East (Washington, D.C.)' ),
 	ams: __( 'EU West (Amsterdam, Netherlands)' ),
 } );
+
+export const getPlanName = () => ( {
+	[ DotcomPlans.BUSINESS ]: __( 'Business' ),
```

Comment:
Dumb question, how can we be sure that these will be translated in the context of WordPress.com plan names, and not just a random English word "Business" 🤔

Reply:
How about naming it getPlanNames? Since it's returning a plan name object rather than taking a plan slug and returning a single plan name 🤔